server/asset/dcr/dcr.go

```diff
@@ -264,6 +264,46 @@ func (dcr *Backend) CheckAddress(addr string) bool {
 	return err == nil
 }
 
+// CoinDetails gets the recipient address, value, and confirmations of a
+// transaction output encoded by the given coinID. If the output does not exist
+// or has a pubkey script of the wrong type, an error will be returned.
+func (dcr *Backend) CoinDetails(coinID []byte) (addr string, val uint64, confs int64, err error) {
+	txHash, vout, errCoin := decodeCoinID(coinID)
+	if errCoin != nil {
+		err = fmt.Errorf("error decoding coin ID %x: %v", coinID, errCoin)
+		return
+	}
+	return dcr.OutputSummary(txHash, vout)
+}
+
+// OutputSummary gets the recipient address, value, and confirmations of a
+// transaction output. If the output does not exist or has a pubkey script of
+// the wrong type, an error will be returned.
+func (dcr *Backend) OutputSummary(txHash *chainhash.Hash, vout uint32) (addr string, val uint64, confs int64, err error) {
+	var verboseTx *chainjson.TxRawResult
+	verboseTx, err = dcr.node.GetRawTransactionVerbose(txHash)
+	if err != nil {
+		if isTxNotFoundErr(err) {
+			err = asset.CoinNotFoundError
+		}
```

Comment:
This PR is still draft because these could use some unit test coverage, but anyone who has experienced issues with notify_fee not responding, please do try this out.

Reply:
This also does not constrain the pkscript of this output nearly enough to ensure that the dex can actually redeem it.  Will update that too.